### PR TITLE
NextJs Pwa Code Example

### PR DESCRIPTION
**Pull request by sharozraees802**
## Bug

- [x] Related issues linked using `fixes #number`
- [x] Integration tests added

## Feature

- [x] Implements an existing feature request or RFC. Make sure the feature request has been accepted for implementation before opening a PR.
- [x] Related issues linked using `fixes #number`
- [x] Integration tests added
- [x] Documentation added
- [x] Telemetry added. In case of a feature if it's used or not.

## Documentation / Examples

- [x] Make sure the linting passes

### CODE REVIEW

File: examples/next_pwa_app/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+## Pwa in next Js
+
+This is a [Next-Pwa](https://www.npmjs.com/package/next-pwa) install and the create a next.Config.js file add Code
+
+
+
+```code
```
Comment (hsh2001) [inline examples/next_pwa_app/README.md:7]:
```suggestion
```js
```

before: 
```code
console.log(this.name);
```

after:
```js
console.log(this.name);
```

File: examples/next_pwa_app/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+## Pwa in next Js
```
Comment (hsh2001) [inline examples/next_pwa_app/README.md:1]:
```suggestion
## PWA in next Js
```

File: examples/next_pwa_app/README.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+This is a [Next-Pwa](https://www.npmjs.com/package/next-pwa) install and the create a next.Config.js file add Code
```
Comment (hsh2001) [inline examples/next_pwa_app/README.md:3]:
```suggestion
This is a [next-pwa](https://www.npmjs.com/package/next-pwa) install and the create a next.Config.js file add Code
```

The package name is `next-pwa` in lowercase right?

**Review by hsh2001 — CHANGES_REQUESTED** [review]
Nice example! Please see my suggestion.

**Review by balazsorban44 — CHANGES_REQUESTED** [review]
Can we bring this example up-to-date with the [examples guidelines](https://github.com/vercel/next.js/blob/canary/contributing/examples/adding-examples.md)? If not, that is understandable, let's close it for now and revisit later on. :pray: :green_heart: Thanks for the work either way!

**Comment by balazsorban44** [comment]
Closing as we actually already have an example: https://github.com/vercel/next.js/blob/canary/examples/progressive-web-app

Which might need deprecation: https://github.com/vercel/next.js/issues/43439

**Pull request closed without merging**